spring-integration-mail/src/main/java/org/springframework/integration/mail/config/MailReceiverFactoryBean.java

```diff
@@ -185,6 +195,9 @@ private MailReceiver createReceiver() {
 		}
 		receiver.setMaxFetchSize(this.maxFetchSize);
 		receiver.setSelectorExpression(selectorExpression);
+		if (this.userFlag != null) {
```

Comment:
Out of topic: `createReceiver()` method here has typo in one of the `Assert` messages.